chart/compass/charts/director/templates/tenant-loader-job.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
if configmap does not exist, this job will fail, so there should be the same condition as for configmap creation